### DISCUSSION TITLE
Template for Good First Issues

### DISCUSSION THREAD

**Discussion by Azaya89** [bokeh/bokeh#13670] [Documentation]
I would like to revisit our recent discussion item regarding the limited availability of `Good First Issues` suitable for new contributors.

I understand that there is an existing template for creating `Good First Issues`.  I think it would be beneficial to conduct a review of existing `Issues` using the template to identify and categorize those that align with the criteria for `Good First Issues`, and possibly create new ones as well.

@bokeh/dev

**Comment by bryevdv** [comment]
For reference here was the previous issue: https://github.com/bokeh/bokeh/issues/11353

And the resulting template: https://github.com/bokeh/bokeh/wiki/Good-first-issue-template

I haven't done a careful review of issues in some time but I guess I'd be surprised if any of the existing issues would make great GFI material. I think it is more likely we will have to purposefully craft some new issues for things that would be reasonable for someone to do, but that we can live with no-one doing them for a potentially open-ended amount of time. 

It might be nice to have one GFI for a docs task, a python task, and maybe a BokehJS task

**Comment by Azaya89** [comment]
I have gone through the template and it seems there are not any current Issues I can see that fit the tag. However, it seems this issue #11765 still has a few files that can be worked on by new contributors. They still qualify as docs tasks and I think they can be worked on still while we think up some more GFIs?

**Comment by bryevdv** [comment]
To be honest I would personally like to close/retire that metadata issue. It has so much history and replies now that it is hard to navigate, but it's also actually not a great first issue in a couple of ways, namely the format for references is obscure and undocumented, and also contributors aren't actually able to see their results on all the non-gallery examples. I think it served a good purpose for some previous group efforts but maybe is not ideal for a 1-1 encounter with a new contributor outside a group activity.
